src/actions/teams/teams.ts

```diff
@@ -0,0 +1,138 @@
+import * as httpRequest from "request-promise-native"
+import * as winston from "winston"
+import * as Hub from "../../hub"
+
+export class TeamsAction extends Hub.Action {
+  name = "teams_incomingwebhook"
+  label = "Teams - Incoming Webhook"
+  iconName = "teams/teams.png"
+  description = "Send data to Teams Incoming webhook"
+  supportedActionTypes = [Hub.ActionType.Query, Hub.ActionType.Dashboard]
+  supportedFormats = [Hub.ActionFormat.Csv, Hub.ActionFormat.WysiwygPng]
+  supportedFormattings = [Hub.ActionFormatting.Unformatted]
+  supportedVisualizationFormattings = [
+    Hub.ActionVisualizationFormatting.Noapply,
+  ]
+  params = []
+
+  async execute(req: Hub.ActionRequest) {
+    let response = { success: true, message: "success" }
+
+    if (!req.formParams.webhookUrl) {
```

Comment:
will the webhook for teams be changing on a frequent basis and will business users have knowledge of this endpoint? if not, it'd be good to make the webhook part of the action set up for example like this: https://github.com/looker/actions/blob/82767e2170b11ec53bac4c37e9a16e3a43906fa4/src/actions/marketo/marketo.ts#L10

the error should pass back to Looker in an Action response e.g. https://github.com/looker/actions/blob/82767e2170b11ec53bac4c37e9a16e3a43906fa4/src/actions/marketo/marketo_transaction.ts#L139